ext/dba/tests/bug78808.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Bug #78808 ([LMDB] MDB_MAP_FULL: Environment mapsize limit reached)
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+$handler = 'lmdb';
+require_once __DIR__ .'/skipif.inc';
+?>
+--FILE--
+<?php
+$handler = 'lmdb';
+require_once __DIR__ .'/test.inc';
```

Comment:
Need add for lmdb:
```
@unlink($db_filename.'-lock');
```

Reply:
That should be handled by the `--CLEAN--` section.